pkg/managerclient/restore.go

```diff
@@ -4,21 +4,25 @@ package managerclient
 
 // Stage enumeration.
 const (
-	RestoreStageInit       = "INIT"
-	RestoreStageDisableTGC = "DISABLE_TGC"
-	RestoreStageData       = "DATA"
-	RestoreStageRepair     = "REPAIR"
-	RestoreStageEnableTG   = "ENABLE_TGC"
-	RestoreStageDone       = "DONE"
+	RestoreStageInit          = "INIT"
+	RestoreStageDropViews     = "DROP_VIEWS"
+	RestoreStageDisableTGC    = "DISABLE_TGC"
+	RestoreStageData          = "DATA"
+	RestoreStageRepair        = "REPAIR"
+	RestoreStageEnableTG      = "ENABLE_TGC"
+	RestoreStageRecreateViews = "RECREATE_VIEWS"
+	RestoreStageDone          = "DONE"
```

Comment:
Enums in go are handled differently.
https://go.dev/play/p/6a3t9StJIup

Reply:
It is an logical enumeration, as the order of those strings define the order of performed restore stages.
I'm not gonna lie, I added this comment just so that linter does not complain, so I don't mind changing it to something that you think would be more fitting.